frontend/public/components/machine-autoscaler.tsx

```diff
@@ -41,11 +41,11 @@ const MachineAutoscalerTargetLink: React.FC<MachineAutoscalerTargetLinkProps> =
 };
 
 const tableColumnClasses = [
-  classNames('col-md-4', 'col-sm-4', 'col-xs-6'),
-  classNames('col-md-3', 'col-sm-4', 'col-xs-6'),
-  classNames('col-md-3', 'col-sm-4', 'hidden-xs'),
-  classNames('col-md-1', 'hidden-sm', 'hidden-xs'),
-  classNames('col-md-1', 'hidden-sm', 'hidden-xs'),
+  'pf-u-w-33-on-lg',
+  '',
+  'pf-m-hidden pf-m-visible-on-md',
+  'pf-m-hidden pf-m-visible-on-lg pf-u-w-8-on-lg',
+  'pf-m-hidden pf-m-visible-on-md pf-u-w-8-on-lg',
```

Comment:
```suggestion
  'pf-m-hidden pf-m-visible-on-lg pf-u-w-16-on-lg',
  'pf-m-hidden pf-m-visible-on-md pf-u-w-16-on-lg',
```
with `pf-u-w-8-on-lg` :
![image](https://user-images.githubusercontent.com/9964343/122551774-11893600-d053-11eb-9c0a-11112df00bfc.png)

after changing to `pf-u-w-16-on-lg`:
![image](https://user-images.githubusercontent.com/9964343/122552592-20bcb380-d054-11eb-819f-99e16f7fbcc0.png)

Reply:
updated